user/src/main/java/org/dicadeveloper/weplantaforest/projects/Price.java

```diff
@@ -37,6 +41,7 @@
     private ScontoType scontoType = ScontoType.NONE;
 
     @Column(name = "_amount", precision = 7, scale = 2)
+    @JsonView({Views.Project.class})

```

Comment:
This means, that the view project, contains articles, which contains a price with an amount.
It's the same structure like the entities/tables are defined.
With this View interface you can only define which elements(columns/references) should be visible when calling the url, you can't change the structure of the entity or their references to other entities.

You define the View by adding/removing  @JsonView({Views.Project.class}) to the elements you want to show with this View, so there is no explicit definition of it.